spec/money_spec.rb

```diff
@@ -43,6 +43,12 @@
     expect(non_fractional_money.to_s(:legacy_dollars)).to eq("1.00")
   end
 
+  it "to_s with_currency style" do
+    expect(amount_money.to_s(:with_currency)).to eq("1.23 USD")
+    expect(non_fractional_money.to_s(:with_currency)).to eq("1.0 JPY")
```

Comment:
I guess, since this line is for a non-factional money, would be nicer to have `"1 JPY"` as an output for `non_factional_money.to_s(:with_currency)`